ingest/annotations.py

```diff
@@ -32,13 +32,12 @@ def __init__(
             self.study_id = ObjectId(study_id)
         if study_file_id is not None:
             self.study_file_id = ObjectId(study_file_id)
-        self.file, self.file_handle = self.open_file(
-            file_path, open_as='dataframe', header=[0, 1]
-        )
-        self.headers = self.file.columns.get_level_values(0)
-        self.annot_types = self.file.columns.get_level_values(1)
         # lambda below initializes new key with nested dictionary as value and avoids KeyError
         self.issues = defaultdict(lambda: defaultdict(lambda: defaultdict(list)))
+        csv_file, self.file_handle = self.open_file(self.file_path)
+        # Remove white spaces, quotes, and lowercase
+        self.headers = [header.strip().strip('\"').lower() for header in next(csv_file)]
+        self.annot_types = [type.strip().strip('\"').lower() for type in next(csv_file)]
```

Comment:
I'm happy to see us ignore accidental whitespace in the SCP headers, it avoids a failure mode  that is hard to detect by eye.